src/monodroid/jni/embedded-assemblies.cc

```diff
@@ -81,31 +89,32 @@ EmbeddedAssemblies::get_assembly_data (uint8_t *data, uint32_t data_size, [[mayb
 	if (header->magic == COMPRESSED_DATA_MAGIC) {
 		if (XA_UNLIKELY (compressed_assemblies.descriptors == nullptr)) {
 			log_fatal (LOG_ASSEMBLY, "Compressed assembly found but no descriptor defined");
-			exit (FATAL_EXIT_MISSING_ASSEMBLY);
+			abort ();
```

Comment:
Why `abort()` and not `exit()`?  I forget what `abort()` *does* on Android…

Reply:
`exit()` causes the application to restart, `abort()` kills it with a native `SIGABRT` stack trace

Reply:
There are a lot of places that users complain about a "white screen", and it ended up their app was crashing and restarting.

Should we audit all of the `exit()` calls?

Reply:
Yes, we should, I've been meaning to do it for quite a while now. I'll open a separate PR for this.

Reply:
PR #7734 opened